library/coretests/tests/floats/f128.rs

```diff
@@ -257,114 +247,107 @@ fn test_classify() {
 }
 
 #[test]
-#[cfg(not(miri))]
 #[cfg(target_has_reliable_f128_math)]
```

Comment:
Ideally this would become `any(miri, target_has_reliable_f128_math)` since f16/f128 math is always reliable with Miri... not sure if that's worth the churn though.

Reply:
Might as well since I'm updating those configs anyway :) Done in a fixup